clang/test/Driver/sycl-target-mismatch.cpp

```diff
@@ -22,4 +22,7 @@
 // RUN: %clangxx -fsycl -fsycl-targets=spir64_gen %S/Inputs/SYCL/liblin64.a \
 // RUN:   -Wno-sycl-target -### %s 2>&1 \
 // RUN:  | FileCheck %s -check-prefix=SPIR64_DIAG
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen %S/Inputs/SYCL/liblin64.a \
+// RUN:   -fsycl-force-target=spir64 -### %s 2>&1 \
+// RUN:  | FileCheck %s -check-prefix=SPIR64_DIAG
 // SPIR64_DIAG-NOT: linked binaries do not contain expected
```

Comment:
The check string seems to end abruptly. I think the actual diagnostic is:
linked binaries do not contain expected target; found targets: <found-target-name>
Is it possible to update the check?

Reply:
Sure, I will update the text.